### DISCUSSION TITLE
DIIP V3 specifications

### DISCUSSION THREAD

**Discussion by k00ij** [FIDEScommunity/DIIP#15] [General]
During DIIP meetup #7 (24-1-2024) following profile updates are scheduled for DIIP V3. The DIIP V3 release is scheduled for 31-7-2024. DIIP V3 specs will be "frozen" on 31-3-2024. Until then, details of the V3 specs can be discussed in this Github and during the meetups.

Updates compared to DIIP V2:

Additional credential formats: 
- SD JWT VC
- JSON-LD

Addtional signature types: 
- ES256k
- EcdsaSecp256k1Signature2019
- EcdsaSecp256r1Signature2019

OpenID for Verifiable Credential Issuance:
- upgraded WG draft

OpenID for Verifiable Presentations
- upgrade to new WG draft

SIOP V2
- upgrade to new WG draft

DIF Presentation Exchange
- upgrade to new WG draft

**Comment by JelleMillenaar** [comment]
@k00ij Can you update this with the latest decisions so we can discuss them here?

**Reply by k00ij** [reply]
@JelleMillenaar This is what we discussed in last week's DIIP meetup#9 (28-3-2024). 

- OpenID for Verifiable Credential Issuance Implementer's Draft 1
 https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0-ID1.html
- OpenID for Verifiable Presentations Implementers draft x --> @nklomp to provide link to suggested draft version
- SIOP V2 --> @nklomp to provide link to suggested version
- DIF Presentation Exchange V2.0 --> @nklomp to provide link to suggested version
- SD-JWT VC DM --> @nklomp to provide link to suggested version
- DID methods: DID:WEB and DID:JWK
- Signature types ES256
- Revocation method: OAuth Status List 
- https://drafts.oauth.net/draft-ietf-oauth-status-list/draft-ietf-oauth-status-list.html
- --> @nklomp to provide link to suggested version

**Comment by TimoGlastra** [comment]
Proposal:

- [OpenID for Verifiable Credential Issuance Implementer's Draft 1](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0-ID1.html)
- [OpenID for Verifiable Presentations - Draft 20](https://openid.net/specs/openid-4-verifiable-presentations-1_0-20.html)
- [SIOP V2 - Draft 13](https://openid.net/specs/openid-connect-self-issued-v2-1_0-13.html)
- [DIF Presentation Exchange V2.0.0](https://identity.foundation/presentation-exchange/spec/v2.0.0/)
- [SD-JWT VC Draft 4](https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-04.html)
- DID methods: `did:web` and `did:jwk`
- Signature types ES256
- Revocation method: [OAuth Status List - Draft 2](https://www.ietf.org/archive/id/draft-ietf-oauth-status-list-02.html)

**Comment by ThierryThevenet** [comment]
Great profile !

**Comment by JelleMillenaar** [comment]
We mention [SD-JWT VC Draft 4](https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-04.html). However, that spec is a superset over [SD-JWT](https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/12/). The link inside SD-JWT VC mentions draft-10 of the SD-JWT format, but the link is not pinned and is currently on draft-12. 

> [11.1. ](https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-04.html#section-11.1)[Normative References](https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-04.html#name-normative-references)
[I-D.ietf-oauth-selective-disclosure-jwt]
Fett, D., Yasuda, K., and B. Campbell, "Selective Disclosure for JWTs (SD-JWT)", Work in Progress, Internet-Draft, draft-ietf-oauth-selective-disclosure-jwt-**10**, 13 June 2024, <https://datatracker.ietf.org/api/v1/doc/document/draft-ietf-oauth-selective-disclosure-jwt/>.

So do we go with draft-10 or draft-12 or something else? Does anyone know what makes the most sense here (BTW, perhaps the differences are so minor that it doesn't matter).
